### PR TITLE
Feat: add 4wl.commands.whitelist param to zoo.cfg

### PR DESCRIPTION
**Pull request by fidanf**
Hello

I've made this change to allow a quick and efficient one-liner documented in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,14 @@ See <https://github.com/ansible/ansible/issues/71528> for more information.
 - The ZooKeeper service can be started via: `systemctl start zookeeper`
 - The ZooKeeper service can be stopped via: `systemctl stop zookeeper`
 
+## Which zookeeper is the leader ?
```
Comment (sleighzy) [inline README.md]: Could rephrase and give more context to the command being run. For example:

## Four Letter Word Commands

ZooKeeper can use commands based on four letter words, see <https://zookeeper.apache.org/doc/current/zookeeperAdmin.html#sc_4lw>

The below example uses the `stat` command to find out which instance is the leader.

```bash
for i in 1 2 3 ; do
  echo "zookeeper0$i is a "$(echo stat | nc zookeeper0$i 2181 | grep ^Mode | awk '{print $2}'); 
done
```

**Review by sleighzy — CHANGES_REQUESTED** [review]
Looks good, just a minor update to the README.

File: README.md (hunk 1 above)
Comment (fidanf) [reply README.md]: Done!

Comment (fidanf) [reply README.md]: How do we solve this markdown lint tyranny ? :|

Comment (sleighzy) [reply README.md]: You referring to the linting rules around line length etc? Do you use an editor like VSCode? Markdown linting extensions should show violations, and formatting/auto-format on save actions will resolve.

**Review by sleighzy — APPROVED** [review]
LGTM. Thanks @fidanf

**Pull request merged**